src/lib.rs

```diff
@@ -23,6 +24,9 @@ pub mod bindings {
 #[doc(inline)]
 pub use bindings::CanPlayTypeResult as CanPlayType;
 pub use top::GeckoMedia;
+pub use top::finish_tests;
```

Comment:
It's a bit weird to expose these 3 things in the public API :/

Reply:
Yeah, they need to be pub in order for the linker to find them when linking the C++ callers when linking the final library.